components/Header.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <header>
+    <div class="hero-content container mx-auto px-4">
+      <div class="hero-text-left">
+        <h1 class="text-4xl">Lorem ipsum</h1>
+        <p class="text-base">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias aperiam, at distinctio doloremque doloribus esse facere impedit inventore laborum modi mollitia natus nesciunt nihil nostrum odit porro quibusdam quidem reiciendis repellendus soluta sunt ut voluptatem.</p>
+        <p class="text-base">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque dolorem iusto, nisi quas quia tempore unde? Assumenda dolores expedita officiis rerum! Adipisci architecto atque debitis earum eos esse in nam?</p>
+        <Button @click="goToLink()" text="Înregistrează-te acum!" />
+      </div>
+      <div class="hero-video-right">
+        <iframe width="100%" height="100%" src="https://www.youtube.com/embed/GEgSBuYlSoA" />
+      </div>
+    </div>
+  </header>
+</template>
+
+<script>
+export default {
+  methods: {
+    goToLink (e) {
+      location.href = '#register'
+    }
+  }
+}
+</script>
+<style>
+
+  header {
+    background: url("static/bg.webp");
+    position: relative;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: rgb(70, 52, 78);
+    width: 100%;
+    height: 100%;
```

Comment:
100vh si 100vw

sau, cand o sa le muti in tailwind, o sa fie w-screen si h-screen